server_test.sh

```diff
@@ -64,3 +64,5 @@ curl -X POST \
   -H 'Authorization: Basic amRvZV8zQGVtYWlsLmNvbTph' \
   -H 'cache-control: no-cache'
 done
+
+// Comment
```

Comment:
Have you found any other way to test HTTP APIs besides curl?

Reply:
In most cases I'm using Postman. When ever I need more flexibility I'm using python (requests) or go (see client.go) to write the scripts that access the API.
Since I'm linux guy I'm not scared to use curl for same easy script written in bash.

Reply:
What about `testing` go package?

Reply:
I'm using testing package whenever I need unit testing. And you're right in this case set of tests that ensure that the handlers behave the way it should is good idea.